components/overflow-group/overflow-group-mixin.js

```diff
@@ -129,15 +131,20 @@ export const OverflowGroupMixin = superclass => class extends LocalizeCoreElemen
 			}
 		});
 
+		const containerStyles = {
+			minHeight: `${this._itemHeight}px`,
+			maxHeight: `${this._itemHeight}px`
+		};
+
 		return html`
-			<div class="d2l-overflow-group-container">
+			<div class="d2l-overflow-group-container" style="${styleMap(containerStyles)}">
 				<slot @slotchange="${this._handleSlotChange}"></slot>
 				${overflowContainer}
 			</div>
 		`;
 	}
 
-	update(changedProperties) {
+	async update(changedProperties) {
```

Comment:
Why was this change needed?

Reply:
Ah forgot to remove this. I'd added it when trying to get something to work.